src/components/skills-quiz/SearchJobCard.jsx

```diff
@@ -1,78 +1,97 @@
-import React, { useContext } from 'react';
+import React, {
+  useContext, useState, useEffect, useMemo,
+} from 'react';
 import PropTypes from 'prop-types';
 import Truncate from 'react-truncate';
 import Skeleton from 'react-loading-skeleton';
-// TODO: Uncomment this line when jobs are coming as hits from Algolia
-// import { camelCaseObject } from '@edx/frontend-platform/utils';
 import { Card } from '@edx/paragon';
 import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
 
-const SearchJobCard = ({ hit, isLoading }) => { // eslint-disable-line no-unused-vars
+const SearchJobCard = ({ index }) => {
   const { refinements } = useContext(SearchContext);
-  const { skill_names: skills } = refinements;
+  const { name: jobs } = refinements;
```

Comment:
can we please define `name` in constants and get it from there at all places. Since we are planning to change it in the future, so it is possible we can miss it at someplace if we are defining it.

Reply:
I tried it this way but Javascipt doesn't let you destructure an object like this, it's returning me `undefined`

```javascript 
const JOB_NAME_ATTRIBUTE = 'name';
const { JOB_NAME_ATTRIBUTE: jobs } = refinements;
```